aws-enigma-2/main.tf

```diff
@@ -0,0 +1,205 @@
+variable "aws_heroku_org" {}
+
+variable "env" {
+  default = "enigma"
+}
+
+variable "github_users" {}
+
+variable "index" {
+  default = 2
+}
+
+variable "rabbitmq_password_org" {}
+
+variable "rabbitmq_username_org" {}
+
+variable "syslog_address_org" {}
+
+data "aws_ami" "tfw" {
+  most_recent = true
+
+  filter {
+    name   = "tag:role"
+    values = ["tfw"]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  owners = ["self"]
+}
+
+variable "amethyst_image" {
+  default = "travisci/ci-amethyst:packer-1512508255-986baf0"
+}
+
+variable "garnet_image" {
+  default = "travisci/ci-garnet:packer-1512502276-986baf0"
+}
+
+terraform {
+  backend "s3" {
+    bucket         = "travis-terraform-state"
+    key            = "terraform-config/aws-enigma-2.tfstate"
+    region         = "us-east-1"
+    encrypt        = "true"
+    dynamodb_table = "travis-terraform-state"
+  }
+}
+
+provider "aws" {
+  version = "~> 1.4"
+}
+
+provider "heroku" {
+  version = "0.1.0"
+}
+
+data "terraform_remote_state" "vpc" {
+  backend = "s3"
+
+  config {
+    bucket         = "travis-terraform-state"
+    key            = "terraform-config/aws-shared-2.tfstate"
+    region         = "us-east-1"
+    dynamodb_table = "travis-terraform-state"
+  }
+}
+
+resource "random_id" "cyclist_token_org" {
```

Comment:
Does this pool need cyclist, or can we run the ASG at a fixed scale?

Reply:
Can we do both? I.e. have cyclist, but run at a fixed scale, like in staging?

Reply:
If it's a fixed size, meaning there are no scaling policies, then there should be no scaling lifecycle events being published, which means that cyclist is sitting there doing nothing :grin:

Reply:
I was thinking it would be preferable to have a cyclist instance doing nothing (at least at first) than refactoring code so it works without cyclist, especially since we might want scaling at some point.
(This is partially driven by laziness, because I don't know how much work it would be to remove the dependency on cyclist, or to add it back later. And also partially a desire for parity among pools.)